gazelle/manifest/copy_to_source.sh

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env bash
```

Comment:
This does not support Windows. Since we are in `rules_python` what do you think about rewriting this as a `py_binary`? I think the original `go_binary` method supported Windows?

Reply:
Rewritten in Python!